lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -51,12 +53,16 @@ def create_access_token
           client
         )
 
-        @access_token = AccessToken.create!(
+        create_params = {
           application_id: refresh_token.application_id,
+          expires_in: expires_in,
+          previous_refresh_token: refresh_token.refresh_token,
```

Comment:
The `refresh_token` attribute might be nil, but `previous_refresh_token` doesn't allow for that. We need to add `.to_s`: `previous_refresh_token: refresh_token.refresh_token.to_s,`.